### PR TITLE
Distinguish output device

### PR DESCRIPTION
**Pull request by Nesurion**
If the output is not a terminal (ie. redirected output to file, email ...), the spinner will not be shown.

#420

### CODE REVIEW

**Comment by Nesurion** [comment]
@marians @ZeissS what do you think? LGTY?

File: main.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package main
 
 import (
 	"fmt"
+	"golang.org/x/crypto/ssh/terminal"
```
Comment (ZeissS) [inline main.go]: I think we need to add new libaries to the Makefile.

Comment (Nesurion) [inline main.go]: sry forgot about that. done.

**Comment by ZeissS** [comment]
@marians Wanna give it a spin? :dancers:

**Comment by marians** [comment]
Looks good!

_Yay! You just unlocked new features._

**Comment by marians** [comment]
@Nesurion Ready for merging?

**Comment by ewoutp** [comment]
LGTM

**Comment by ewoutp** [comment]
WFM

**Comment by marians** [comment]
Merging this after having talked to @Nesurion

**Pull request merged**